app/src/main/java/org/oppia/android/app/customview/ContinueButtonView.kt

```diff
@@ -116,9 +118,15 @@ class ContinueButtonView @JvmOverloads constructor(
   }
 
   private fun startAnimating() {
-    val animation = AnimationUtils.loadAnimation(context, R.anim.scale_button_size)
+    val animation = AnimationUtils.loadAnimation(context, R.anim.wobble_button_animation)
     if (enableContinueButtonAnimation.value) {
       startAnimation(animation)
+      // Repeat the animation after a fixed interval.
+      lifecycleSafeTimerFactory.createTimer(INTERVAL_BETWEEN_CONTINUE_BUTTON_ANIM_MS).observe(
+        fragment
+      ) {
+        startAnimating()
+      }
```

Comment:
```suggestion
      lifecycleSafeTimerFactory.createTimer(
        INTERVAL_BETWEEN_CONTINUE_BUTTON_ANIM_MS
      ).observe(fragmen)) {
        startAnimating()
      }
```

Generally prefer line-breaking at the higher syntactical element.

That also being said, does just passing in ``::startAnimating`` instead of the lambda work?

Reply:
Added the suggestive commit. Re: ::startAnimating, this isn't the lambda but the recursive function itself; Am i missing something?

Reply:
The method reference is just a syntax sugar alternative to the lambda (it'll result in generating the observer implementation). Consider resolving this optional, but a nice-to-have.

Reply:
![image](https://user-images.githubusercontent.com/64526117/215169068-ce821522-131b-43eb-b547-c9e6e6dc47af.png)
This would be the way to do it, and in my opinion, this looks a bit more complex. WDYT? I'll change as per the review if  you still suggest so!

Reply:
Ah, no that's not what I was thinking, and is definitely less readable than the current code. I was instead thinking something like this:

```kotlin
lifecycleSafeTimerFactory.createTimer(
  INTERVAL_BETWEEN_CONTINUE_BUTTON_ANIM_MS
).observe(fragment, ::startAnimating)
```

This only works if startAnimating matches in signature to the Observer class SAM. I'm not sure it will in which case we should just keep the current code. If it does, the above is a bit cleaner.

Reply:
I'm getting a type mismatch error while doing this. Lets go with the current implementation.

Reply:
Sounds good--thanks for checking!